addons/mail/models/mail_message.py

```diff
@@ -68,6 +70,7 @@ def _get_default_author(self):
     mail_activity_type_id = fields.Many2one(
         'mail.activity.type', 'Mail Activity Type',
         index=True, ondelete='set null')
+    is_internal = fields.Boolean('Employee Only')
```

Comment:
I don't know if it's necessary because it seems to always be given on create, but maybe the default value should be True for security reasons?

Reply:
For old db records, subtype.internal will still set old messages as internal (still supported in acls checks) and will avoid a migration. For new records normally value should be set correctly by caller (message_something).

Reply:
Then I feel it would more clear if it was commented on the field `is_internal` that `subtype.internal` still matters and that `is_internal` is used as an additional AND during security check and not replacing it.

Reply:
Or maybe a better name for the field would be `force_internal` instead? True forces it, False does nothing.